app/Content/src/Http/Requests/BlogPosts/BlogPostTagsRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Content\Http\Requests\BlogPosts;
+
+use App\Platform\Enums\RoleName;
+use Illuminate\Foundation\Http\FormRequest;
+
+class BlogPostTagsRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     */
+    public function authorize(): bool
+    {
+        return $this->user()->hasRole(RoleName::ADMIN) || $this->user()->hasRole(RoleName::CONTENT_MANAGER);
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array<string, \Illuminate\Contracts\Validation\ValidationRule|array<mixed>|string>
+     */
+    public function rules(): array
+    {
+        return [
+           'name' => ['required']
```

Comment:
The blog post tag name should also be a string and have a maximum of 255 characters.

Reply:
Also, tags should be unique.